### PR TITLE
Push changes to Main

### PR DESCRIPTION
**Pull request by afreen910**
1. Module to pull data from HTTP endpoint
2. Module to export data to a csv file

### CODE REVIEW

File: get_user.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+URL = "https://reqres.in/api/users"
+
+
+class Users:
+
+    def get_page_count(self, url):
+        response = requests.get(url)
+        page_count = response.json()['total_pages']
+        return page_count
+
```
Comment (roshanzameer) [inline get_user.py:27]: Since you are making a request here please add a Try-Except block.

File: get_user.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # # Add handlers to the logger
+    # logger.addHandler(c_handler)
+    # logger.addHandler(f_handler)
+
+    logger.info('get_user.py user is the main caller')
+    from csv_export import export_to_csv
+
+    user_object = Users()
+    result = user_object.get_all_data(URL)
+    print(result)
```
Comment (roshanzameer) [inline get_user.py]: _export_to_csv_ imported but not being called. Please make changes.

**Review by roshanzameer — CHANGES_REQUESTED** [review]
Changes mentioned inline

**Comment by afreen910** [comment]
@roshanzameer New changes have been added. Please review

**Review by roshanzameer — APPROVED** [review]
LGTM!

**Pull request merged**